install/kubernetes/helm/meshery/charts/meshery-traefik-mesh/README.md

```diff
@@ -36,10 +35,11 @@ Meshery Adapter for Traefik Mesh chart.
 | resources | object | `{}` |  |
 | securityContext | object | `{}` |  |
 | service.annotations | object | `{}` |  |
-| service.port | int | `10000` |  |
+| service.port | int | `10006` |  |
```

Comment:
Ouch! This might be causing some of the trouble that you're seeing with Traefik Mesh, @asubedy.

Reply:
Here we have the AutoMake tool on our Makefile. And any change on Helm charts need to run the Automake script. And we can avoid these type of issues.